<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WMRawActualConfigBase</class>
 <widget class="QDialog" name="WMRawActualConfigBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vektoren Formatsteuerung</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QGroupBox" name="AmplitudebuttonGroup">
       <property name="title">
        <string>Amplitude</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QRadioButton" name="Amplx1radioButton">
          <property name="text">
           <string>x 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="Amplx1_sqrt2radioButton">
          <property name="text">
           <string>x 1/SQRT(2)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="primsekbuttonGroup">
       <property name="title">
        <string>prim/sek</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QRadioButton" name="PrimradioButton">
          <property name="text">
           <string>primär</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="SekradioButton">
          <property name="text">
           <string>sekundär</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="WinkelbuttonGroup">
       <property name="title">
        <string>Winkel</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QRadioButton" name="MathradioButton">
          <property name="text">
           <string>math. positiv</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="TechradioButton">
          <property name="text">
           <string>techn. positiv</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin" stdset="0">
      <number>0</number>
     </property>
     <item>
      <spacer name="Horizontal Spacing2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>WMRawActualConfigBase</receiver>
   <slot>SendData2AppSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>WMRawActualConfigBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>WMRawActualConfigBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SendData2AppSlot()</slot>
  <slot>amplx1checkedSlot()</slot>
  <slot>amplx1_sq2checkedSlot()</slot>
  <slot>winkelmathcheckedSlot()</slot>
  <slot>winkeltechcheckedSlot()</slot>
  <slot>primCheckedSlot()</slot>
  <slot>sekCheckedSlot()</slot>
 </slots>
</ui>
